packages/bsky/src/hydration/hydrator.ts

```diff
@@ -1022,6 +1064,10 @@ export const mergeStates = (
     labelerAggs: mergeMaps(stateA.labelerAggs, stateB.labelerAggs),
     labelerViewers: mergeMaps(stateA.labelerViewers, stateB.labelerViewers),
     knownFollowers: mergeMaps(stateA.knownFollowers, stateB.knownFollowers),
+    bidirectionalBlocks: mergeMaps(
+      stateA.bidirectionalBlocks,
+      stateB.bidirectionalBlocks,
+    ),
```

Comment:
The way `bidirectionalBlocks` works, I think this would allow some clobbering to happen.  For example, how should these two merge?

```
{ did:example:a -> { did:example:b -> true } }
{ did:example:a -> { did:example:c -> true } }
```

I think we'd want to find something like this:
```
{ did:example:a -> { did:example:b -> true, did:example:c -> true } }
```